00-zero-reconcile/zr.py

```diff
@@ -0,0 +1,47 @@
+import time # used to wait
+
+expected = {
+	"key": "00000000"
+}
+
+current = expected.copy()
+
+def main():
+	counter = 0
+	print('Key set: ' + current.get('key'))
+	while True:
```

Comment:
I would consider just a single run through the loop, instead of two state checks per loop, but its nitpicky and not super important to the demo of the concept.

The name "wellness" doesn't give me enough info on what the function does. A descriptive name like `compare_actual_expected_states`  is more useful for human readability.

```
		wellness()
		time.sleep(3)
		counter += 1
		if counter == 3:
			current['key'] = '11111111'
			counter = 0
```